hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/block/TestDeletedBlockLog.java

```diff
@@ -360,50 +371,78 @@ public void testResetCount() throws Exception {
     // Increment another time so it exceed the maxRetry.
     // On this call, count will be set to -1 which means TX eventually fails.
     incrementCount(txIDs);
-    blocks = getTransactions(40 * BLOCKS_PER_TXN);
+    blocks = getTransactions(40 * BLOCKS_PER_TXN * THREE);
     for (DeletedBlocksTransaction block : blocks) {
       Assertions.assertEquals(-1, block.getCount());
     }
 
     // If all TXs are failed, getTransactions call will always return nothing.
-    blocks = getTransactions(40 * BLOCKS_PER_TXN);
+    blocks = getTransactions(40 * BLOCKS_PER_TXN * THREE);
     Assertions.assertEquals(0, blocks.size());
 
     // Reset the retry count, these transactions should be accessible.
     resetCount(txIDs);
-    blocks = getTransactions(40 * BLOCKS_PER_TXN);
+    blocks = getTransactions(40 * BLOCKS_PER_TXN * THREE);
     for (DeletedBlocksTransaction block : blocks) {
       Assertions.assertEquals(0, block.getCount());
     }
-    Assertions.assertEquals(30, blocks.size());
+    Assertions.assertEquals(90, blocks.size());
```

Comment:
It looks like most of these tests are not testing the throttling of the service so the number passed in to `getTransactions` is not relevant as long as it's sufficiently large, right? Could we
- Pass Integer.MAX_VALUE to `getTransactions` when limiting the number of returned results is not important. Maybe a wrapper in the test like `getAllTransactions` which does this would be more intuitive.
- Use the THREE replication constant as a multiple in the assert statements.

There's also room to improve these tests in general by replacing the magic numbers with variables and asserting the number of transactions returned instead of commenting the expected behavior, but since this PR is not changing that part directly we don't have to make such a sweeping change.

Reply:
> It looks like most of these tests are not testing the throttling of the service so the number passed in to getTransactions is not relevant as long as it's sufficiently large, right? 

Some cases related to committing certain num of txns also need the replication constant so I perserved it

> Pass Integer.MAX_VALUE to getTransactions when limiting the number of returned results is not important. Maybe a wrapper in the test like getAllTransactions which does this would be more intuitive.

Done.

> Use the THREE replication constant as a multiple in the assert statements. 

Done